content/en/containers/guide/operator-advanced.md

```diff
@@ -20,6 +20,7 @@ Using the Datadog Operator requires the following prerequisites:
 
 ## Deploy the Datadog Operator
 
+To facilitate the command execution, we recommend defining an environment variable called `DD_NAMESPACE` in your shell.
```

Comment:
```suggestion
To facilitate the command execution, define an environment variable called `DD_NAMESPACE` in your shell.
```
Since the instructions below rely on the existence of the variable, I think it's most straightforward to just tell the user to set it up.